packages/api/models/user.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+const {
+  Model
+} = require('sequelize');
```

Comment:
This feels like a lot of space for one import. What do you think of just doing it on one line, like `const { Model } = require('sequelize');`?